source/tests/gperf_cpu_tests.cpp

```diff
@@ -128,8 +128,22 @@ allocate(int64_t nfactor)
     printf("fibonacci(%li) * %li = %li\n", (long) nfib, (long) niter, ret);
 }
 
+#if defined(TIMEMORY_WINDOWS)
```

Comment:
You can just call `tim::set_env(...)`

https://github.com/NERSC/timemory/blob/05495bc3516d0a94aef7b40827df4ccc495b0a95/source/timemory/environment/declaration.hpp#L234

Reply:
I changed the cmake logic so that gperf tests are not built on windows, so I should not have checked in this change. I will push a revert soon.